integration_tests/integrations/rails/.gitignore

```diff
@@ -0,0 +1 @@
+vendor/bundle
```

Comment:
Ah, it already has a vendor directory that's ignored - https://github.com/alphagov/govuk_template/blob/master/integration_tests/integrations/mustache/vendor/.gitignore